blocksync/blocksync_test.go

```diff
@@ -555,8 +555,8 @@ func TestBlockSyncerPeerBlockList(t *testing.T) {
 		nil,
 	)
 
-	require.NoError(bs.ProcessBlock(ctx, "peer1", blk2))
-	require.NoError(bs.ProcessBlock(ctx, "peer2", blk1))
+	require.NoError(bs.ProcessBlock(ctx, "peer1", blk1))
+	require.NoError(bs.ProcessBlock(ctx, "peer2", blk2))
```

Comment:
trying to understand why we need this PR?

the testing is passing on latest code, and I guess process blk2 before blk1 is intentionally?

Reply:
Yes, it's intentionally. I add some comments on original code for quick understanding

Reply:
ok, so I think we don't need this PR any more?

or, your change tested more cases for this blocksync test?